android/jni/com/mapswithme/maps/Sponsored.cpp

```diff
@@ -100,13 +100,11 @@ JNIEXPORT void JNICALL Java_com_mapswithme_maps_widget_placepage_Sponsored_nativ
   string const hotelId = jni::ToNativeString(env, id);
   string const code = jni::ToNativeString(env, currencyCode);
 
-  g_framework->RequestBookingMinPrice(hotelId, code, [hotelId](string const & price,
-                                                               string const & currency) {
-    GetPlatform().RunOnGuiThread([=]() {
-      JNIEnv * env = jni::GetEnv();
-      env->CallStaticVoidMethod(g_sponsoredClass, g_priceCallback, jni::ToJavaString(env, hotelId),
-                                jni::ToJavaString(env, price), jni::ToJavaString(env, currency));
-    });
+  g_framework->RequestBookingMinPrice(hotelId, code, [=](string const & price,
+                                                         string const & currency) {
+    JNIEnv * env = jni::GetEnv();
+    env->CallStaticVoidMethod(g_sponsoredClass, g_priceCallback, jni::ToJavaString(env, hotelId),
```

Comment:
Do we really need to capture g_sponsoredClass and g_priceCallback by copy? As I can see, only these globals are captured from the outer context.